modules/vpc/metadata.yaml

```
@@ -19,121 +19,146 @@ metadata:
-    location: examples/private_service_connect_google_apis
-  - name: routes
-    location: examples/routes
-  - name: secondary_ranges
-    location: examples/secondary_ranges
-  - name: simple_project
-    location: examples/simple_project
-  - name: simple_project_with_regional_network
-    location: examples/simple_project_with_regional_network
-  - name: submodule_firewall
-    location: examples/submodule_firewall
-  - name: submodule_network_peering
-    location: examples/submodule_network_peering
-  - name: submodule_svpc_access
-    location: examples/submodule_svpc_access
-  - name: submodule_vpc_serverless_connector
-    location: examples/submodule_vpc_serverless_connector
-  variables:
-  - name: auto_create_subnetworks
-    description: When set to true, the network is created in 'auto subnet mode' and it will create a subnet for each region automatically across the 10.128.0.0/9 address range. When set to false, the network is created in 'custom subnet mode' so the user can explicitly connect subnetwork resources.
-    type: bool
-    default: false
-    required: false
-  - name: delete_default_internet_gateway_routes
-    description: If set, ensure that all routes within the network specified whose names begin with 'default-route' and with a next hop of 'default-internet-gateway' are deleted
-    type: bool
-    default: false
-    required: false
-  - name: description
-    description: An optional description of this resource. The resource must be recreated to modify this field.
-    type: string
-    default: ""
-    required: false
-  - name: mtu
-    description: 'The network MTU (If set to 0, meaning MTU is unset - defaults to ''1460''). Recommended values: 1460 (default for historic reasons), 1500 (Internet default), or 8896 (for Jumbo packets). Allowed are all values in the range 1300 to 8896, inclusively.'
-    type: number
-    default: 0
-    required: false
-  - name: network_name
-    description: The name of the network being created
-    type: string
-    required: true
-  - name: project_id
-    description: The ID of the project where this VPC will be created
-    type: string
-    required: true
-  - name: routing_mode
-    description: The network routing mode (default 'GLOBAL')
-    type: string
-    default: GLOBAL
-    required: false
-  - name: shared_vpc_host
-    description: Makes this project a Shared VPC host if 'true' (default 'false')
-    type: bool
-    default: false
-    required: false
-  outputs:
-  - name: network
-    description: The VPC resource being created
-  - name: network_id
-    description: The ID of the VPC being created
-  - name: network_name
-    description: The name of the VPC being created
-  - name: network_self_link
-    description: The URI of the VPC being created
-  - name: project_id
-    description: VPC project id
-  roles:
-  - level: Project
+  info:
+    title: Terraform VPC Module
+    source:
+      repo: https://github.com/q2w/terraform-google-network.git
```

nit: here and a few places
```suggestion
      repo: https://github.com/terraform-google-modules/terraform-google-network.git
```

---

Thanks! Done.